src/test/java/edu/harvard/iq/dataverse/DatasetFieldServiceBeanFilterResponseTest.java

```diff
@@ -0,0 +1,186 @@
+//        rootLogger.setLevel(Level.FINE);
+//
+//        // Ensure all handlers respect the FINE level
+//        for (var handler : rootLogger.getHandlers()) {
+//            handler.setLevel(Level.FINE);
+//        }
+//
+//        // Add a ConsoleHandler if none exists
+//        if (rootLogger.getHandlers().length == 0) {
+//            ConsoleHandler consoleHandler = new ConsoleHandler();
+//            rootLogger.addHandler(consoleHandler);
+//        }
+    }
+
+    @AfterEach
+    void tearDown() {
+      this.datasetFieldServiceBean = null;
+    }
+
+    @Test
+    void audience() throws Exception {
+        String termURI = "https://www.narcis.nl/classification/D13700";
+        JsonObject cvocEntry = readObject("src/test/resources/json/cvoc-dans-config/audience.json");
+        JsonObject readObject = readObject("src/test/resources/json/cvoc-dans-value/audience.json");
+
+        JsonObject result = callFilterResponse(cvocEntry, readObject, termURI);
+
+        assertThat(result.getString("@id")).isEqualTo(termURI);
+        assertThat(getValue("nl", result.get("vocabularyName"))).startsWith("Classificatiecodes van de wetenschapsportal Narcis (www.narcis.nl).");
+        assertThat(getValue("nl", result.get("termName"))).startsWith("Theoretische chemie, kwantumchemie");
+        assertThat(result.keySet()).containsExactlyInAnyOrder("@id", "termName", "vocabularyUri", "vocabularyName");
+    }
+
+    @Test
+    void collection() throws Exception {
+        String termURI = "https://vocabularies.dans.knaw.nl/collections/archaeology/ArcheoDepot";
+        JsonObject cvocEntry = readObject("src/test/resources/json/cvoc-dans-config/collection.json");
+        JsonObject readObject = readObject("src/test/resources/json/cvoc-dans-value/collection.json");
+
+        JsonObject result = callFilterResponse(cvocEntry, readObject, termURI);
+
+        assertThat(result.getString("@id")).isEqualTo(termURI);
+        assertThat(getValue("nl", result.get("vocabularyName"))).isEqualTo("DansCollections");
+        assertThat(getValue("nl", result.get("termName"))).isEqualTo("ArcheoDepot");
+        assertThat(result.keySet()).containsExactlyInAnyOrder("@id", "termName", "vocabularyUri", "vocabularyName");
+    }
+
+    @Test
+    void aatClassification() throws Exception {
+        String termURI = "https://vocab.getty.edu/aat/300187008";
+        JsonObject cvocEntry = readObject("src/test/resources/json/cvoc-dans-config/AATClassification.json");
+        JsonObject readObject = readObject("src/test/resources/json/cvoc-dans-value/AATClassification.json");
+
+        JsonObject result = callFilterResponse(cvocEntry, readObject, termURI);
+
+        assertThat(result.getString("@id")).isEqualTo(termURI);
+        assertThat(getValue("en", result.get("vocabularyName"))).isEqualTo("The Art and Architecture Thesaurus Concepts");
+        assertThat(result.getString("termName")).isEqualTo("Abnakee kleden");
```

Comment:
test fails because termName is not found.

Reply:
Based on the config file, and json value file, I'd expect the value to be ```
[
        {
          "lang": "nl",
          "value": "Abnakee kleden"
        },
        {
          "lang": "en",
          "value": "Abnakee rugs"
        }
      ]
```